lnbits/static/js/components.js

```diff
@@ -356,6 +356,24 @@ Vue.component('lnbits-lnurlpay-success-action', {
   }
 })
 
+Vue.component('lnbits-qrcode', {
```

Comment:
Do we really want to name the component `lnbits-qrcode`? 

This has a drawback that all extension would need to be updated to use the new element.

Can't we just name this component as `qrcode` and get all extension QR code with LNbits logo updated for free?

Reply:
they need to be changed in steps anyway, using `qrcode` as the name conflicts with the VueQRcode library, at to what I can figure out (i may be doing something wrong though)! But no all qrcodes need to change, some are wrapped in a `<q-responsive />` , some are not...